### PR TITLE
Stop abilities from being disabled after level 5000

### PR DESCRIPTION
**Pull request by DanielMcAssey**
Fix for #103

### CODE REVIEW

**Comment by xPaw** [comment]
```
[00:21:44]<esw> why not, say after level 5000 or so, start disabling nuke use EXCEPT on bosses?
[00:22:16]<esw> also by nukes I mean any of the high dps items like napalm and cluster bomb and such
```

**Comment by DanielMcAssey** [comment]
So:

```
if(!targetIsTreasureOrBoss && getGameLevel() > 5000) {
    disableAbility(ABILITIES.NUKE);
}
```

Something like that then?

**Comment by EienSeihai** [comment]
Disable cluster bomb and napalm as well, they're far more useful in the boss levels as well. 

Also, does the script still automate nuke (and the other two) uses? If so that needs to be disabled as well unless disableAbility handles that.

**Comment by DanielMcAssey** [comment]
I believe disableAbility handles that.

**Comment by EienSeihai** [comment]
Then this should be fine, discuss with other devs to make sure that they agree.

File: autoPlay.user.js [hunk 1]
```diff
@@ -826,6 +825,12 @@ function goToLaneWithBestTarget() {
 			// Throw Money at Screen
 			enableAbilityItem(ITEMS.THROW_MONEY);
 		}
+		
+		if(!targetIsTreasureOrBoss && getGameLevel() >= 5000) {
```
Comment (xPaw) [inline autoPlay.user.js]: Pretty sure it doesn't need to be an extra `if`, and can just go into previous `else`

**Comment by DanielMcAssey** [comment]
Just to show an example of a level >5000

No Boss:
no_boss

Boss:
with_boss

**Pull request closed without merging**